src/s1reader/s1_reader.py

```diff
@@ -459,12 +461,22 @@ def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str = 'vv'):
     if not os.path.exists(path):
         raise FileNotFoundError(f'{path} not found')
     elif os.path.isdir(path):
-        return _burst_from_safe_dir(path, id_str, orbit_path)
+        bursts = _burst_from_safe_dir(path, id_str, orbit_path)
     elif os.path.isfile(path):
-        return _burst_from_zip(path, id_str, orbit_path)
+        bursts = _burst_from_zip(path, id_str, orbit_path)
     else:
         raise ValueError(f'{path} is unsupported')
 
+    # ensure burst IDs is a list
+    if not isinstance(burst_ids, list) and isinstance(burst_ids, str):
+        burst_ids = [burst_ids]
+
+    if burst_ids:
+        return [b for b in bursts if b.burst_id in burst_ids]
```

Comment:
I wonder if we should provide some useful informations here before returning. There would be cases that all the requested burst_ids are found in the zip file and returned. Also there will be situations that only part of the requested burst_ids are found in the zip file or maybe even None of them found. I think it will be very helpful to print/log some useful information on which burst_id is found and which one was not found before returning the list.